src/showcase/app/shared/component-viewer/stackblitz-writer/stackblitz-writer.service.ts

```diff
@@ -0,0 +1,207 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+export class ExampleData {
+  indexFilename: string;
+  description: string;
+  selectorName: string;
+  componentName: string;
+  exampleFiles: { name: string; content: string }[];
+  business: boolean;
+
+  constructor(data: any) {
+    this.indexFilename = data.indexFilename;
+    this.description = data.description;
+    this.selectorName = data.selectorName;
+    this.componentName = data.componentName;
+    this.exampleFiles = data.exampleFiles;
+    this.business = data.business;
+  }
+}
+
+const STACKBLITZ_URL = 'https://run.stackblitz.com/api/angular/v1';
+
+/**
+ * Path that refers to the docs-content from the "@angular/components-examples" package.
+ */
+const CONTENT_PATH = 'assets/docs-content/examples-source/';
+
+/**
+ * Path that refers to the generic project template files.
+ */
+const TEMPLATE_PATH = '/src/showcase/assets/stackblitz/';
```

Comment:
This should be `assets/stackblitz` to be compliant with the base href.
This probably relates to the comment from @jeripeierSBB.